README.rst

```diff
@@ -0,0 +1,180 @@
+.. ## Define a hard line break in HTML
+.. |br| raw:: html
+
+  <br />
+
+.. # Add the reference for the badges
+.. ## Reference to the CI status
+.. |Linter&Tests| image:: https://github.com/mind-inria/hidimstat/actions/workflows/ci.yml/badge.svg?branch=main
```

Comment:
This badge does not render well

Reply:
It's normal because the name of the workflow is not yet modified.
It will be done with this PR.